[1/5] Performing build step for 'bootloader'
[1/1] cd /home/matiasnd/esp-workspace/Esp32_rtos/build/bootloader/esp-idf/esptool_py && /home/matiasnd/.espressif/python_env/idf5.2_py3.10_env/bin/python /home/matiasnd/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 bootloader 0x1000 /home/matiasnd/esp-workspace/Esp32_rtos/build/bootloader/bootloader.bin
Bootloader binary size 0x6900 bytes. 0x700 bytes (6%) free.
[2/3] cd /home/matiasnd/esp-workspace/Esp32_rtos/build/esp-idf/esptool_py && /home/matiasnd/.espressif/python_env/idf5.2_py3.10_env/bin/python /home/matiasnd/esp/esp-idf/components/partition_table/check_sizes.py --offset 0x8000 partition --type app /home/matiasnd/esp-workspace/Esp32_rtos/build/partition_table/partition-table.bin /home/matiasnd/esp-workspace/Esp32_rtos/build/app-template.bin
app-template.bin binary size 0x30570 bytes. Smallest app partition is 0x100000 bytes. 0xcfa90 bytes (81%) free.
[2/3] cd /home/matiasnd/esp/esp-idf/components/esptool_py && /home/matiasnd/.espressif/tools/cmake/3.24.0/bin/cmake -D IDF_PATH=/home/matiasnd/esp/esp-idf -D "SERIAL_TOOL=/home/matiasnd/.espressif/python_env/idf5.2_py3.10_env/bin/python;;/home/matiasnd/esp/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/matiasnd/esp-workspace/Esp32_rtos/build -P /home/matiasnd/esp/esp-idf/components/esptool_py/run_serial_tool.cmake
esptool.py --chip esp32 -p /dev/ttyUSB0 -b 460800 --before=default_reset --after=hard_reset write_flash --flash_mode dio --flash_freq 40m --flash_size 2MB 0x1000 bootloader/bootloader.bin 0x10000 app-template.bin 0x8000 partition_table/partition-table.bin
esptool.py v4.7.dev2
Serial port /dev/ttyUSB0
Connecting....

A serial exception error occurred: device reports readiness to read but returned no data (device disconnected or multiple access on port?)
Note: This error originates from pySerial. It is likely not a problem with esptool, but with the hardware connection or drivers.
For troubleshooting steps visit: https://docs.espressif.com/projects/esptool/en/latest/troubleshooting.html
FAILED: CMakeFiles/flash /home/matiasnd/esp-workspace/Esp32_rtos/build/CMakeFiles/flash 
cd /home/matiasnd/esp/esp-idf/components/esptool_py && /home/matiasnd/.espressif/tools/cmake/3.24.0/bin/cmake -D IDF_PATH=/home/matiasnd/esp/esp-idf -D "SERIAL_TOOL=/home/matiasnd/.espressif/python_env/idf5.2_py3.10_env/bin/python;;/home/matiasnd/esp/esp-idf/components/esptool_py/esptool/esptool.py;--chip;esp32" -D "SERIAL_TOOL_ARGS=--before=default_reset;--after=hard_reset;write_flash;@flash_args" -D WORKING_DIRECTORY=/home/matiasnd/esp-workspace/Esp32_rtos/build -P /home/matiasnd/esp/esp-idf/components/esptool_py/run_serial_tool.cmake
ninja: build stopped: subcommand failed.
